### ISSUE TITLE
Support for other Telink based devices?

### ISSUE THREAD

**Issue by giggo** [devbis/z03mmc#48]
I flashed the firmware to a [Xiaomi Mijia BLE MHO-C401](https://pvvx.github.io/MHO_C401/) and Zigbee works fine.  The display obviously doesn't. Do you have plans to support the other devices that are supported in @pvvx 's firmware (https://github.com/pvvx/ATC_MiThermometer)?

**Comment by devbis** [comment]
Yes, I have such plans, but it's not a top priority since I don't have such devices. :) 
Does temperature and humidity are detected correctly?

**Comment by pvvx** [comment]
> Are temperature and humidity detected correctly?

Defective or fake sensors are installed on variants С401 from Azarton. The data is sent to a different I2c address (0x00).

**Comment by pvvx** [comment]
Added CGDK2 in [TelinkMiFlasher.html](https://pvvx.github.io/ATC_MiThermometer/TelinkMiFlasher.html)

**Comment by pvvx** [comment]
Added test version MHO-C401(N), where N is release from 2022.
MHO-C401 have 3 global hardware modifications...
![image](https://github.com/devbis/z03mmc/assets/12629515/a2839c3a-c0af-4b8a-8d99-1e7097a81a61)
